react-native-in-app-payments-quickstart/app/components/OrderModal.tsx

```diff
@@ -28,23 +28,23 @@ import AddressView from './AddressView';
 import GreenButton from './GreenButton';
 import DigitalWalletButton from './DigitalWalletButton';
 
-OrderModal.propTypes = {
-  onCloseOrderScreen: PropTypes.func.isRequired,
-  onPayWithGiftCard: PropTypes.func.isRequired,
-  onPayWithCard: PropTypes.func.isRequired,
-  onShowDigitalWallet: PropTypes.func.isRequired,
-  onBuyerVerification: PropTypes.func.isRequired,
+interface OrderModal {
+  onCloseOrderScreen: any,
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.